lstchain/io/io.py

```diff
@@ -1018,7 +1025,46 @@ def read_dl2_to_pyirf(filename):
     events = pd.read_hdf(filename, key=dl2_params_lstcam_key).rename(columns=name_mapping)
     events = table.QTable.from_pandas(events)
 
+    #Make the columns as Quantity
```

Comment:
Slightly Offtopic:
Are there any code style guidelines enforced in lstchain?
Ctapipe has a black pre commit hook, that would fix some of the style things (Whitespace after #).

Reply:
Yes, lstchain follows the same code style guidelines as ctapipe.
I had forgotten to take it into account while writing my scripts. Thank you for pointing it out! I will soon update accordingly.